src/types.ts

```diff
@@ -252,10 +252,9 @@ export interface UIDisputeTokens {
 }
 
 export interface StakeDetails extends NormalizedPayout {
-  totalStake: string;
-  completedStake: string;
+  stakeCompleted: string;
   size?: string;
-  currentStake?: string;
+  stakeCurrent?: string;
   tentativeWinning: boolean;
```

Comment:
also stakeCurrent, I don't see a reason for it to be nullable, it would just b '0'

Reply:
Again, tentativewinning